packages/block-library/src/table/editor.scss

```diff
@@ -6,6 +6,7 @@
 		height: auto;
 
 		table {
+			table-layout: fixed;
```

Comment:
Isn't this unnecessary since `style.scss` is loaded in the editor?

Reply:
In testing it was for me, feel free to iterate on PR if you don't find it is in testing @ZebulanStanphill